src/main/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/notification/MediationSuccessfulApplicantNotificationHandler.java

```diff
@@ -45,15 +54,83 @@ public String camundaActivityId(CallbackParams callbackParams) {
 
     private CallbackResponse notifyApplicant(CallbackParams callbackParams) {
         CaseData caseData = callbackParams.getCaseData();
-        notificationService.sendMail(
-            caseData.getApplicantSolicitor1UserDetails().getEmail(),
-            notificationsProperties.getNotifyApplicantLRMediationSuccessfulTemplate(),
-            addProperties(caseData),
-            String.format(REFERENCE_TEMPLATE, caseData.getLegacyCaseReference()));
+        Boolean isCarmEnabled = featureToggleService.isCarmEnabledForCase(caseData.getSubmittedDate());
+        if (isCarmEnabled) {
+            String claimId = caseData.getCcdCaseReference().toString();
+            String referenceTemplate = String.format(REFERENCE_TEMPLATE, claimId);
+            String application1Email = caseData.getApplicantSolicitor1UserDetails().getEmail();
```

Comment:
this will work for LR but if applicant is a lip then we should read from caseData.getApplicant1().getPartyEmail()